tests/test_cases/pipeonestring.c

```diff
@@ -4,7 +4,7 @@
 #include <unistd.h>
 #include <string.h>
 
-#define BUFSIZE 16
+#define BUFSIZE 10
```

Comment:
I believe I fixed this one recently. I'm worried its reverting?

Reply:
This must be an old change as I haven't touched the file, I'll fix it.